api/controllers/admin.js

```diff
@@ -11,7 +12,8 @@ module.exports = {
 };
 
 function config(req, res) {
-  res.json(req.app.get("config"));
+  var redact_passwords = Scrubbers.object_keys(["password"]);
```

Comment:
Mostly it's just because that's what the sample code looked like. Author of the library is a coffeescript dev, so there are likely different (and more ruby-ish) style guidelines for that. But both of your suggestions sound good.